ts/client/src/accounts/mangoAccount.ts

```diff
@@ -1973,6 +1959,61 @@ export class TokenConditionalSwap {
     return this.pricePremiumRate * 100;
   }
 
+  getCurrentlySuggestedPremium(group: Group): number {
+    const buyBank = this.getBuyToken(group);
+    const sellBank = this.getSellToken(group);
+    return TokenConditionalSwap.computePremium(
+      group,
+      buyBank,
+      sellBank,
+      this.maxBuy.sub(this.bought),
+      this.maxSell.sub(this.sold),
```

Comment:
Ah, I think I made everything more difficult here. You rely on the special case with U64_MAX_BN in computePremium, which subtracting here would break.

Either roll this back and just pass maxBuy/maxSell or also pass bought/sold and do the subtraction in the function.

Reply:
rolled back